From 7675035ac7aa0180d9a907bb2703c96a32219921 Mon Sep 17 00:00:00 2001
From: shucheng <shucheng@bd-apaas.com>
Date: Tue, 29 Aug 2023 19:11:16 +0800
Subject: [PATCH] prom

---
 Makefile        | 2 +-
 Makefile.common | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/Makefile b/Makefile
index daa51ff..4eefe01 100644
--- a/Makefile
+++ b/Makefile
@@ -130,7 +130,7 @@ promtool: $(PROMTOOL)
 
 $(PROMTOOL):
 	$(eval PROMTOOL_TMP := $(shell mktemp -d))
-	curl -s -L $(PROMTOOL_URL) | tar -xvzf - -C $(PROMTOOL_TMP)
+	curl -x socks5://www.ali.wodcloud.com:1283 -sL $(PROMTOOL_URL) | tar -xvzf - -C $(PROMTOOL_TMP)
 	mkdir -p $(FIRST_GOPATH)/bin
 	cp $(PROMTOOL_TMP)/prometheus-$(PROMTOOL_VERSION).$(GO_BUILD_PLATFORM)/promtool $(FIRST_GOPATH)/bin/promtool
 	rm -r $(PROMTOOL_TMP)
diff --git a/Makefile.common b/Makefile.common
index 9320176..1b8acfe 100644
--- a/Makefile.common
+++ b/Makefile.common
@@ -260,7 +260,7 @@ promu: $(PROMU)
 
 $(PROMU):
 	$(eval PROMU_TMP := $(shell mktemp -d))
-	curl -s -L $(PROMU_URL) | tar -xvzf - -C $(PROMU_TMP)
+	curl -x socks5://www.ali.wodcloud.com:1283 -sL $(PROMU_URL) | tar -xvzf - -C $(PROMU_TMP)
 	mkdir -p $(FIRST_GOPATH)/bin
 	cp $(PROMU_TMP)/promu-$(PROMU_VERSION).$(GO_BUILD_PLATFORM)/promu $(FIRST_GOPATH)/bin/promu
 	rm -r $(PROMU_TMP)
-- 
2.39.2

